### PR TITLE
Refactor tracing example

### LINKED ISSUE #56: Improve consistency of example "example-tracing"
**Description**
The example has some inconsistencies which should be fixed:
- Adjust naming by renaming to "tracing"
- Adjust location of yaml file by having it in "deployment" folder and a better naming of the file
- Move go sources into a "src" folder

**Reasons**
Consistency across all examples to increase developer experience

### PR DESCRIPTION
**Pull request by lilitgh**
**Description**

Changes proposed in this pull request:

- Refactor tracing example

**Related issue(s)**
Resolves #56

### CODE REVIEW

File: tracing/deployment/deployment.yaml [hunk 1]
```diff
@@ -46,7 +46,7 @@ spec:
     spec:
       containers:
         - name: order-front
-          image: eu.gcr.io/kyma-project/example/tracing/order-front:0.0.1
```
[the remark this answers is not in the document]
Comment (lilitgh) [reply tracing/deployment/deployment.yaml]: renamed it to tracing-order-front

File: tracing/README.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+- Kyma as the target deployment environment.
+- An Environment created in Kyma to which you deploy the example application.
+- Helm for local installation.
+
+
+## Installation
+
+### Locall Installation
+
+If you use a local Deployment of Kyma on Minikube, you need to know that Jaeger installation is optional, and you cannot install it locally by default. However, you can install it on a Kyma instance and run it locally using Helm.
```
[the remark this answers is not in the document]
Comment (a-thaler) [reply tracing/README.md]: I created a ticket to document correct approach.

File: tracing/README.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+1. Export your Environment as variable by replacing the `{environment}` placeholder in the following command and running it:
+
+    ```bash
+    export KYMA_EXAMPLE_ENV="{environment}"
+    ```
+
+2. Change the value of the **hostname** parameter from `order-front-api.kyma.local` to `order-front-api.{YOUR_CLUSTER_DOMAIN}` in the [`deployment`](deployment/deployment.yaml) file.
+
+3. Deploy the service. Run this command:
```
Comment (bszwarc) [inline tracing/README.md]: Should we add the directory you run the command from? Like: 
```
In the` tracing/examples` directory, run the following command:
```
or is it too obvious and redundant from a dev point of view :D

Comment (lilitgh) [reply tracing/README.md]: I think we don't need it here :)

Comment (bszwarc) [reply tracing/README.md]: Ok, then let's leave it as it is :)

File: tracing/README.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+
+> **NOTE:** If you use a local Deployment of Kyma on Minikube,  be aware that Jaeger installation is optional, and you cannot install it locally by default. However, you can install it on a Kyma instance and run it locally using Helm.
+
+1. To install Jaeger, go to the [Kyma resources](https://github.com/kyma-project/kyma/tree/master/resources) directory and run the following command:
+
+```bash
+helm install -n jaeger -f jaeger/values.yaml --namespace kyma-system --set-string global.domainName=kyma.local --set-string global.isLocalEnv=true jaeger/
+```
+
+2. Follow the instructions in  **cluster installation** section (skip step 2). You can access the tracing UI locally at `https://jaeger.kyma.local`.
```
Comment (bszwarc) [inline tracing/README.md]:
```suggestion
2. Follow the instructions in the  **Cluster installation** section (skip step 2). You can access the tracing UI locally at `https://jaeger.kyma.local`.
```

File: tracing/README.md [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+
+### Cluster installation
+
+1. Export your Environment as variable by replacing the `{environment}` placeholder in the following command and running it:
```
Comment (bszwarc) [inline tracing/README.md]:
```suggestion
1. Export your Environment as a variable by replacing the `{environment}` placeholder in the following command and running it:
```

**Pull request merged**